CONTRIBUTING.md

```diff
@@ -53,3 +53,9 @@ VS code doesn't have "Source Root" so you can skip this step if you use VS Code.
 
 ## Tests
 The [README under tests folder](tests/README.md) tells you how to set up to run tests.
+
+## Why do my PR tests fail?
+If this happens to you do not panic! Any PRs originating from a fork will fail some automated tests. This is because
+forks do not have access to our repository's secrets. A maintainer will manually review your changes for malicious
+code then kick off the rest of our testing suite. Feel free to tag one of us if you feel like we are taking too long
```

Comment:
maybe say tag @snowflakedb/snowpark-python-api ?

Reply:
Suggest removing "for malicious code".